UnitedIncome/components
# [APP-2988] Input Explanation Text

### Description
> Please provide a brief description of your changes below.

Adds the `explanation` prop on top of the input field. I've had to make some changes to how the percent field works, but all of them I feel are probably for the best.

![hhzuzuzuz](https://user-images.githubusercontent.com/10888441/59793155-dad18700-92a3-11e9-9196-5fdb9e4a5ce8.gif)


### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Pull down the branch and go into Storybook.
* Find an input field and focus it.
* You'll see the explanation prop in action.
* Also added it to the simple form example within QuestionCard.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
